config-connector/solutions/iam/kpt/pubsub-subscription/Kptfile

```diff
@@ -0,0 +1,36 @@
+apiVersion: kpt.dev/v1alpha1
+kind: Kptfile
+metadata:
+  name: pubsub-subscription
+packageMetadata:
+  shortDescription: configures permissions for a pubsub subscription
```

Comment:
"Create a pubsub subscription and configure permissions for it"?

Reply:
I don't think we want to emphasize the creation of a subscription because IAM is the focus.

Reply:
Could you explain what change have you made and resolve the comment?

Reply:
I have changed the description to reflect that the solution creates a subscription.